src/ui/studio/save-creation/index.tsx

```diff
@@ -39,13 +39,13 @@ import {
 
 import Notification from '../../notification';
 import { ActionButtons } from '../elements';
-import { Input } from '../../elements';
+import { Input, SeriesDropdown, SeriesOption } from '../../elements';
 
 import RecordingPreview, { RecordingPreviewHandle } from './recording-preview';
 
 import { otherShortcuts } from '../../../shortcuts';
 import { StepProps } from '../steps';
-
+import { SingleValue } from 'react-select';
```

Comment:
Ideally this should be in the group with the other external imports, i.e. above all internal imports.